src/s1reader/s1_annotation.py

```diff
@@ -5,15 +5,16 @@
 
 from dataclasses import dataclass
 import datetime
+import os
 import xml.etree.ElementTree as ET
+import zipfile
 
 import numpy as np
 
 from packaging import version
-from scipy.interpolate import InterpolatedUnivariateSpline
 
-#some thresholds
-version_threshold_azimuth_noise_vector=version.parse('2.90')
+#A IPF version from which has azimuth noise vector
+version_threshold_azimuth_noise_vector = version.parse('2.90')
```

Comment:
In fact, it's the version that NADS started to include azimuth noise vector. I can see that the variable's name was not clear enough to make you interpret that other way around.

My suggestions are as below:

```python
#IPF version from which the NADS has azimuth noise vector annotation
ipf_version_az_noise_vector_available_from = version.parse('2.90')
```

Reply:
```python
#IPF version from which the NADS has azimuth noise vector annotation
min_ipf_version_az_noise_vector = version.parse('2.90')
```
I think this could work too.